app/src/main/res/values/colors.xml

```diff
@@ -15,4 +15,5 @@
     <color name="focus_screen_bg">#0a0a0a</color>
     <color name="side_nav_start_color">#6c60a9</color>
     <color name="side_nav_end_color">#7367ee</color>
+    <color name="memory_graph_background">#673AB7</color>
```

Comment:
Why did you mark "No modifications done at the end of resource files strings.xml, dimens.xml or colors.xml" as done?

Reply:
Made these changes while updating. Will unmark from PR body template. Thanks.